shared/constants/tracker.js

```diff
@@ -145,3 +146,12 @@ export function isLoading (state: ?TrackerState): boolean {
 
   return !state.userInfo || state.userInfo.followersCount === -1
 }
+
+export function bufferToNiceHexString (fingerPrint: Buffer): string {
+  try {
+    // $FlowIssue
+    return fingerPrint.toString('hex').slice(-16).toUpperCase().match(/(.{4})(.{4})(.{4})(.{4})/).slice(1).join(' ')
```

Comment:
Hmmm this should work... https://github.com/facebook/flow/blob/master/tests/try/return.js#L14